### ISSUE TITLE
segfault on new turn

### ISSUE THREAD

**Issue by Voker57** [freeorion/freeorion#1409]
Does not appear to be reproducible.

Version: teamsmac-8.7, based on 7bfb87f9583f4dbd8da570.

Save: http://dump.bitcheese.net/files/burasal/l_v_n.mps

OS: Linux debian/sid

Video card: nVidia geForce 1080
CPU: 6770k overclocked to 4400 Mhz, tested stable

Backtrace:

```
#0  0x00007f643b701e9c in UniverseObject::ID (this=0x0) at /home/voker57/Sources/freeorion/universe/UniverseObject.cpp:127
127	{ return m_id; }


#0  0x00007f643b701e9c in UniverseObject::ID (this=0x0) at /home/voker57/Sources/freeorion/universe/UniverseObject.cpp:127
#1  0x00007f643b6064d4 in Fleet::MovePath (this=0x562f0453d930, route=std::__cxx11::list = {...}, flag_blockades=true)
    at /home/voker57/Sources/freeorion/universe/Fleet.cpp:462
#2  0x0000562ef1dc3c03 in MapWnd::SetFleetMovementLine (this=0x562ef33ad730, fleet_id=924)
    at /home/voker57/Sources/freeorion/UI/MapWnd.cpp:4369
#3  0x0000562ef1dc7d93 in MapWnd::DeferredRefreshFleetButtons (this=0x562ef33ad730)
    at /home/voker57/Sources/freeorion/UI/MapWnd.cpp:4729
#4  0x0000562ef1db0932 in MapWnd::PreRender (this=0x562ef33ad730) at /home/voker57/Sources/freeorion/UI/MapWnd.cpp:1613
#5  0x00007f6438042ba5 in GG::GUI::PreRenderWindow (wnd=0x562ef33ad730) at /home/voker57/Sources/freeorion/GG/src/GUI.cpp:1569
#6  0x00007f643804354c in GG::GUI::PreRender (this=0x7ffca8a829c0) at /home/voker57/Sources/freeorion/GG/src/GUI.cpp:1662
#7  0x00007f6437fdb436 in GG::EventPumpBase::LoopBody (this=0x7ffca8a81e70, gui=0x7ffca8a829c0, state=..., do_non_rendering=true, 
    do_rendering=true) at /home/voker57/Sources/freeorion/GG/src/EventPump.cpp:79
#8  0x00007f6437fdb609 in GG::ModalEventPump::operator() (this=0x7ffca8a81e70)
    at /home/voker57/Sources/freeorion/GG/src/EventPump.cpp:114
#9  0x00007f643a5ef3de in GG::SDLGUI::Run (this=0x7ffca8a829c0) at /home/voker57/Sources/freeorion/GG/src/SDL/SDLGUI.cpp:883
#10 0x00007f643804091b in GG::GUI::operator() (this=0x7ffca8a829c0) at /home/voker57/Sources/freeorion/GG/src/GUI.cpp:1090
#11 0x00007f643a5ee2c2 in GG::SDLGUI::operator() (this=0x7ffca8a829c0)
    at /home/voker57/Sources/freeorion/GG/src/SDL/SDLGUI.cpp:530
#12 0x0000562ef19efd37 in mainSetupAndRun () at /home/voker57/Sources/freeorion/client/human/chmain.cpp:285
#13 0x0000562ef19eca63 in main (argc=3, argv=0x7ffca8a82e98) at /home/voker57/Sources/freeorion/client/human/chmain.cpp:102
```

**Comment by geoffthemedio** [comment]
hopefully / might be fixed, or at least worked around to avoid a crash.

**Comment by Vezzra** [comment]
@geoffthemedio, why did you tag this as "works for me"? Apparently you put in a fix that's supposed to address the issue, so "resolved" would be the proper tag...?

**Comment by geoffthemedio** [comment]
I don't know if the possible fix actually resolved anything, but it might have.

**Comment by Vezzra** [comment]
I understand, but even in that case "resolved" is still the proper tag, IMO. "Works for me" is intended for issues that can't be reproduced, which means nothing that could be fixed can be identified, and therefore nothing can be fixed.

Although not being able to reproduce applies here, you apparently still identified a potential cause for the issue described and provided an attempt to fix that. Even if we're not sure if the fix accomplished anything, this issue has been closed on the assumption it did.

So I'm going to change the status to "resolved". If you still think that "works for me" is the proper status, please change that back again.

**Comment by Vezzra** [comment]
Fixed by 248ec50bb69cd660f2335f951c948880c72f997a.
